### PR TITLE
Bump version of costa

### PR DESCRIPTION
**Pull request by mtaillefumier**
update of costa version

### CODE REVIEW

**Review by haampie — APPROVED** [review]

File: var/spack/repos/builtin/packages/costa/package.py [hunk 1]
```diff
@@ -21,6 +21,7 @@ class Costa(CMakePackage):
     # note: The default archives produced with github do not have the archives
     #       of the submodules.
     version("master", branch="master", submodules=True)
+    version("2.1", sha256="c1e86452415083f7470b292d93ec60708b7c8dbafc2bac383636bb4b28135866")
```
Comment (haampie) [inline var/spack/repos/builtin/packages/costa/package.py:24]: Looks like it's missing the release tarball https://github.com/eth-cscs/COSTA/releases

Version 2.0 has it

**Review by haampie — CHANGES_REQUESTED** [review]
doesn't download

**Comment by mtaillefumier** [comment]
I need to pull some strings to fix this. Costa versioning  seems to have changed.

**Comment by mtaillefumier** [comment]
Thanks @haampie. we found a different solution to this problem. I will close this one and someone from the group will open a new one.

**Pull request closed without merging**